### PR TITLE
feat: change a log level while a request to github failed because a commit not found

### PR DESCRIPTION
**Pull request by niktverd**
…ot found

### CODE REVIEW

File: src/vcs-connector/client/github.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import log from '@doc-tools/transform/lib/log';
 import {Octokit} from '@octokit/core';
+import {GraphqlResponseError} from '@octokit/graphql';
```
Comment (3y3) [inline src/vcs-connector/client/github.ts:3]: This dependency in not installed in project

File: src/vcs-connector/client/github.ts [hunk 2]
```diff
@@ -41,7 +42,14 @@ async function getRepoCommitByHash(httpClientByToken: Octokit, hashCommit: strin
 
         return commit.data;
     } catch (error) {
+        if ((error as GraphqlResponseError<{message: string}>)?.response?.data?.message?.includes(
```
Comment (3y3) [inline src/vcs-connector/client/github.ts:45]: Is there more precise class for `instanceof` check?

**Review by 3y3 — CHANGES_REQUESTED** [review]

File: src/vcs-connector/client/github.ts (hunk 1 above)
Comment (niktverd) [reply src/vcs-connector/client/github.ts]: Removed it and replaced with custom class

File: src/vcs-connector/client/github.ts (hunk 2 above)
Comment (niktverd) [reply src/vcs-connector/client/github.ts]: Replaced it with custom class. Error Type in @octokit made confused way

File: src/vcs-connector/client/github.ts [hunk 3]
```diff
@@ -8,6 +8,17 @@ import {
     GithubUserDTO,
 } from '../connector-models';
 import {validateConnectorFields} from '../connector-validator';
+import {logger} from '../../utils';
+
+class ResponseError extends Error {
```
Comment (3y3) [inline src/vcs-connector/client/github.ts:13]: I don't see where this class will be instantiated.
Should it be a Type?

**Review by 3y3 — CHANGES_REQUESTED** [review]

**Review by 3y3 — APPROVED** [review]

**Pull request merged**